Tests/src/PublicAPITests/UserSystemTestHelpers.h

```diff
@@ -33,11 +33,11 @@ void LoadTestAccountCredentials();
 
 void LogIn(csp::systems::UserSystem* UserSystem,
 		   csp::common::String& OutUserId,
-		   const csp::common::String& Email									 = DefaultLoginEmail,
-		   const csp::common::String& Password								 = DefaultLoginPassword,
-		   bool AgeVerified													 = true,
-		   csp::services::EResultCode ExpectedResultCode					 = csp::services::EResultCode::Success,
-		   csp::services::EResultBaseFailureReason ExpectedResultFailureCode = csp::services::EResultBaseFailureReason::None);
+		   const csp::common::String& Email				 = DefaultLoginEmail,
+		   const csp::common::String& Password			 = DefaultLoginPassword,
+		   bool AgeVerified								 = true,
+		   csp::services::EResultCode ExpectedResultCode = csp::services::EResultCode::Success,
+		   int ExpectedResultFailureCode				 = -1);
```

Comment:
-1 is Unknown. The default failure code should be 0, and we should be using an enum here to specify it:

`int ExpectedFailureCode = (int)csp::services::EResultBaseFailureCode::None`

Reply:
sorted